source/assets/stylesheets/_accessibility.scss

```diff
@@ -60,6 +60,11 @@ a:focus {
   outline: 3px solid $focus-colour;
 }
 
+/* Make links slightly darker when focused to improve contrast. */
+a:link:focus {
```

Comment:
What should be the colour when the link is visited? If there is nothing overwriting that, visited links will probably still show the lighter blue.